extensions/markdown-language-features/src/test/pathCompletion.test.ts

```diff
@@ -171,4 +171,14 @@ suite('Markdown path completion provider', () => {
 
 		assert.ok(completions.some(x => x.insertText === 'file.md'), 'Has file from space');
 	});
+
+	test('Should complete path for angled brackets', async () => {
```

Comment:
Please also add tests for:

- Incomplete links:  `[](<${CURSOR}`
- Links that appear inside text on the line: `text [](<${CURSOR}>) more text`
- Links with titles: `[](<${CURSOR}> "title">`
- Any other edge cases

I'm pretty sure the code will break for all of those today